drupal8-example/docker-compose.yml

```diff
@@ -1,36 +1,73 @@
-# Example file for a PHP-based setup.
 version: '3'
-services:
 
+services:
   web:
-    image: reload/drupal-apache-fpm
+    image: reload/drupal-apache-fpm:latest
     ports:
       - '80'
+      - '443'
     depends_on:
-      - php
+      - fpm
+    links:
+      - fpm
     volumes:
       - './web/:/var/www/web:rw,cached'
       - './backend/vendor/:/var/www/vendor:ro,cached'
 
-  php:
+  fpm:
     image: reload/drupal-php7-fpm:7.2
     ports:
       - '9000'
     depends_on:
       - db
+    links:
+      - db
     working_dir: /var/www/web
     volumes:
       - './:/var/www:rw,cached'
+      - './docker/my_init.d/install-cron.sh:/etc/my_init.d/install-cron.sh'
     environment:
       PATH: '/var/www/vendor/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin'
       PHP_SENDMAIL_PATH: /usr/local/bin/mhsendmail --smtp-addr="mailhog:1025"
+      DRUSH_OPTIONS_URI: https://MY-PROJECT.docker
 
   db:
-    image: reload/db-datadir:reloaddk-latest-fulldb
+    image: mariadb:10
     ports:
       - '3306'
+    environment:
```

Comment:
If we use datadirs these environment variables won't do anything. At the very least there should be a comment noting this and maybe describe how to switch to a database-dump.

Reply:
Will fix.